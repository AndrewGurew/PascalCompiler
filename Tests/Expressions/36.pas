foo(1-2, a+b,c) - bar(1+1);