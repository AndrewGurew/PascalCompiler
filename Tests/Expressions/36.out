⎬-
 ⎬foo with params:
  ⎬-
   ⎬1
   ⎬2
  ⎬+
   ⎬a
   ⎬b
  ⎬c
 ⎬bar with params:
  ⎬+
   ⎬1
   ⎬1